Economy.md

```diff
@@ -0,0 +1,93 @@
+# EssentialsX Economy
+The built in economy API will redirect to any other detected economy system such as vault. This means you can use EssEco api as a economy api instead of vault if your plugin is dependent on Essentials.
```

Comment:
IMO the start of the page should explain what the economy is, perhaps something like "EssentialsX comes with a simple, easy-to-use economy system built into the plugin." or similar?

This part about the API should probably be in a separate section at the bottom, since it's only useful to developers and doesn't mean much to most people who want to use EssentialsX.

(Also tiny nitpicks: "vault" should be "Vault", "EssEco" should be "EssentialsX's economy", "api" should be "API")